web/src/app/modules/shared/components/presentation/cytoscape/cytoscape.component.scss

```diff
@@ -16,7 +16,7 @@
   top: 0;
 }
 
-::ng-deep .label1 {
+::ng-deep div .label-header .label1 {
```

Comment:
Nitpicking here but wondering if `.label-header` can be applied inside element which is not a div since cytoscape can change the parent tag for the template. Can this be just `.label-header .label1` and still work fine?

Reply:
Right, good point - clarity 5 was overwriting these in storybook because they were not specific enough, but I went to other extreme and made therm probably too specific.

Reply:
Changed it.